xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -173,27 +174,31 @@
 #define MUSICPLAYER_RATING          219
 #define MUSICPLAYER_COMMENT         220
 #define MUSICPLAYER_LYRICS          221
-#define MUSICPLAYER_HASPREVIOUS     222
-#define MUSICPLAYER_HASNEXT         223
-#define MUSICPLAYER_EXISTS          224
-#define MUSICPLAYER_PLAYLISTPLAYING 225
-#define MUSICPLAYER_ALBUM_ARTIST    226
-#define MUSICPLAYER_PLAYCOUNT       227
-#define MUSICPLAYER_LASTPLAYED      228
-#define MUSICPLAYER_CHANNEL_NAME    229
-// unused 230
-#define MUSICPLAYER_CHANNEL_GROUP   231
-// unused 232
-#define MUSICPLAYER_CHANNEL_NUMBER  233
-#define MUSICPLAYER_CONTENT         234
-#define MUSICPLAYER_USER_RATING     235
-#define MUSICPLAYER_RATING_AND_VOTES     236
-#define MUSICPLAYER_VOTES           237
-#define MUSICPLAYER_MOOD            238
-#define MUSICPLAYER_CONTRIBUTORS    239
-#define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 240
-#define MUSICPLAYER_DBID            241
-#define MUSICPLAYER_PROPERTY        242
+#define MUSICPLAYER_ALBUM_ARTIST    222
+#define MUSICPLAYER_PLAYCOUNT       223
+#define MUSICPLAYER_LASTPLAYED      224
+#define MUSICPLAYER_USER_RATING     225
+#define MUSICPLAYER_RATING_AND_VOTES     226
+#define MUSICPLAYER_VOTES           227
+#define MUSICPLAYER_MOOD            228
+#define MUSICPLAYER_CONTRIBUTORS    229
+#define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 230
+#define MUSICPLAYER_DBID            231
+
+// Range of musicplayer infolabels that work with offset and poistion
```

Comment:
typo: poistion

Reply:
Ah, did that then forgot to save... Think I'll have lunch first

Reply:
bon appetit